README.md

```diff
@@ -208,7 +208,7 @@ Update packages of external applications, e.g. updates that are intended for a t
 
 The boolean option `--zigbee` enables the generation of Zigbee update file in addition to the zip package. The following example demonstrates the generation of such update file:
 ```
-nrfutil pkg generate --hw-version 52 --sd-req 0 --application-version 0x01020101 --application nrf52840_xxaa.hex --key-file ../priv.pem app_dfu_package.zip --zigbee True --manufacturer-id 0xCAFE --image-type 0x1234 --comment good_image
+nrfutil pkg generate --hw-version 52 --sd-req 0 --application-version 0x01020101 --application nrf52840_xxaa.hex --key-file ../priv.pem --app-boot-validation VALIDATE_ECDSA_P256_SHA256 app_dfu_package.zip --zigbee True --zigbee-manufacturer-id 0xCAFE --zigbee-image-type 0x1234 --zigbee-comment good_image --zigbee-ota-hw-version 52 --zigbee-ota-fw-version 0x01020101
```

Comment:
As discussed f2f we need to clarify which version params are truly needed. If parameters are duplicated, eg. ```--hw-version``` and ```--zigbee-ota-hw-version``` then we should simply. It should be described what is the importance of given parameters, i.e., how they should be set.